workers/ohsome_quality_analyst/api/api.py

```diff
@@ -119,8 +130,16 @@ def empty_api_response() -> dict:
     return {
         "apiVersion": __version__,
         "attribution": {
-            "text": "© OpenStreetMap contributors",
-            "url": "https://ohsome.org/copyrights",
+            "text": (
+                "© OpenStreetMap contributors;"
+                "© European Union, 1995-2022"
+                "Global Human Settlement Layer Data;"
+                "Earth Observation Group Nighttime Light Data."
+            ),
+            "url": (
+                "https://github.com/GIScience/ohsome-quality-analyst/blob/main/data/"
+                "LICENSES.md"
+            ),
```

Comment:
Regarding your follow up suggestions: It would be easiest to generate the attribution text on the indicator class level. This would mean the attributions would be an attribute of the resulting GeoJSON Feature Properties.

Should the attribution be at the top level of the response object?

Reply:
I think we should have an attribution `text` field at the top-level as we currently have it. You can (if you want) add further information in the lower levels. How you want to deal with this text is up to you. E.g. having a placeholder text, which is okay if it happens, but never should be transferred to the user, like “Attribution see url field“. Better would be to replace this by the actual sources all the indicators in the response have used.

Reply:
```
#  GhsPopComparison Indicator
{
  "apiVersion": "0.8.0",
  "attribution": {
    "url": "https://github.com/GIScience/ohsome-quality-analyst/blob/main/data/COPYRIGHTS.md",
    "text": "© OpenStreetMap contributors; © European Union, 1995-2022, Global Human Settlement Layer Data."
  },
  ...
}

# MappingSaturation Indicator
{
  "apiVersion": "0.8.0",
  "attribution": {
    "url": "https://github.com/GIScience/ohsome-quality-analyst/blob/main/data/COPYRIGHTS.md",
    "text": "© OpenStreetMap contributors."
  },
    ...
}

# other endpoints (not indicator or report endpoints) including regions endpoint
  {
  "apiVersion": "0.8.0",
  "attribution": {
    "url": "https://github.com/GIScience/ohsome-quality-analyst/blob/main/data/COPYRIGHTS.md"
  },
      ...
}
```

I did not add an attribution text to the regions endpoint. I hope we will not depend on those for long..